SignalServiceKit/src/Contacts/ContactsUpdater.m

```diff
@@ -88,11 +88,11 @@ - (void)lookupIdentifiers:(NSArray<NSString *> *)identifiers
 
     [self contactIntersectionWithSet:[NSSet setWithArray:identifiers]
                              success:^(NSSet<SignalRecipient *> *recipients) {
-                                 if (recipients.count > 0) {
-                                     success(recipients.allObjects);
-                                 } else {
-                                     failure(OWSErrorMakeNoSuchSignalRecipientError());
+                                 if (recipients.count == 0) {
```

Comment:
This condition is an error for lookup `lookupIdentifier` but not for `lookupIdentifier**s**`

Reply:
Actually, this PR isn't ready yet. Some callers expect this list to be none empty... 

Closing for a minute.